src/isPalindrome.js

```diff
@@ -0,0 +1,13 @@
+export default function isPalindrome(inputString){
+
+  inputString = inputString.toLowerCase();
+  inputString = inputString.replace(/[^\w]|_/g, "");
```

Comment:
Could condense this to one line and assign it to a variable that describes what it is:
```js
var inputOnlyLetters = inputString.toLowerCase().replace(/[^\w]|_/g, "");
```